### PR TITLE
Added gallery/camera image fetching utilities

### PR DESCRIPTION
**Pull request by emalamela**
### Summary

This code provides image retrieving utilities through ```GetImageFragment``` implementation which has shortcuts for starting the gallery/camera and retrieving exposing a ```Uri``` of the corresponding image.

In the case of the gallery, it just obtains its ```Uri```, and in the camera case it stores a new file, with name ```GetImageFragment#pictureTakenFilename``` name and then expose its ```Uri```. 

Also, ```FileUtils``` and ```ImageUtils``` were added along by requirement ```GetImageFragment``` usage.

### CODE REVIEW

**Comment by juanignaciomolina** [comment]
ping @emalamela

**Comment by emalamela** [comment]
I'm waiting for @alvarez-fabian to review it. If you are pinging me about the red-crosses, remember that we checked jenkins at the error was not from this branch :slightly_smiling_face:.

File: core/src/main/java/ar/com/wolox/wolmo/core/util/FileUtils.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    private FileUtils() {}
+
+    /**
+     * Creates a file in the external public directory to store data.
+     *
+     * The file ends up being stored as:
+     *  filename + "." + extension
+     *
+     * @param filename File name, used as described above
+     * @param extension ImageFormat of the file, used as described above
```
Comment (alvarez-fabian) [inline core/src/main/java/ar/com/wolox/wolmo/core/util/FileUtils.java]: Please add a javadoc to tell when the `IOException` is thrown.

File: core/src/main/java/ar/com/wolox/wolmo/core/util/ImageUtils.java [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+        if (i.resolveActivity(ContextUtils.getAppContext().getPackageManager()) != null) {
+            File photoFile = null;
+
+            try {
+                photoFile = FileUtils.createFile(filename, format);
+            } catch (IOException ex) {
+                ToastUtils.showToast(errorResId);
+            }
+
+            Uri photoFileUri = Uri.fromFile(photoFile);
```
Comment (alvarez-fabian) [inline core/src/main/java/ar/com/wolox/wolmo/core/util/ImageUtils.java]: NPE Warning! If the file could not be created (IOException is thrown), `photoFile` will be `null` and `Uri.fromFile(photoFile)` will throw a [NPE](https://developer.android.com/reference/android/net/Uri.html#fromFile(java.io.File)).

I think this line should be inside the `if` below.

File: core/src/main/java/ar/com/wolox/wolmo/core/util/ImageUtils.java [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+    /**
+     * Prevents quality from being outside 0...100 range
+     *
+     * @param quality target quality
+     *
+     * @return if below 0, returns 0, if above 100, return 100, else returns {@code quality} param
+     */
+    private static int sanitizeQuality(int quality) {
+        if (quality < 0) return 0;
+        else if (quality > 100) return 100;
```
Comment (alvarez-fabian) [inline core/src/main/java/ar/com/wolox/wolmo/core/util/ImageUtils.java]: Please add some curly braces to this `if` :smile:

File: core/src/main/java/ar/com/wolox/wolmo/core/util/ImageUtils.java [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+        return quality;
+    }
+
+    /**
+     *
```
Comment (alvarez-fabian) [inline core/src/main/java/ar/com/wolox/wolmo/core/util/ImageUtils.java]: Missing description in javadoc

File: core/src/main/java/ar/com/wolox/wolmo/core/util/ImageUtils.java [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+            PNG,
+            JPG
+    })
+    public @interface ImageFormat {
+    }
+
+    public static final String PNG = "png";
+    public static final String JPG = "jpg";
+
+    private ImageUtils() {
```
Comment (alvarez-fabian) [inline core/src/main/java/ar/com/wolox/wolmo/core/util/ImageUtils.java]: Minor thing, but we can put both curly braces together if there is no implementation and no comments. :smile:

**Pull request merged**